src/ui/mails/Mails.js

```diff
@@ -21,8 +24,17 @@ class Mails extends React.Component {
         this._fetchMails();
     }
 
-    _fetchMails() {
-        fetch('/api/mails')
+    filterOnEmailTo(event) {
+        if(event.target.value.length > 3) {
+            this._fetchMails(`to=%${event.target.value}%`);
+        } else if (event.target.value.length < 1) {
+            this._fetchMails();
+        }
```

Comment:
This function is not defined for string lengths of 1, 2 and 3. Could you please make this function exhaustive?

Reply:
Will do it tomorrow!

Reply:
Fixed!

Reply:
👍